commands/raw.go

```diff
@@ -0,0 +1,26 @@
+package commands
+
+// Raw binary data type
+type Raw struct {
+	data []byte
+	node int
+}
+
+// NewRaw creates a new Raw
+func NewRaw(b []byte) *Raw {
+	return &Raw{data: b}
+}
+
+// Encode make sure we implement the Encodable interface
```

Comment:
end in a period. And it would be "makes sure".

But better:

   // Encode implements the Encodable interface.